### ISSUE TITLE
Include example url in examples

### ISSUE THREAD

**Issue by ITCMD** [adnanh/webhook#399]
It would be very helpful if example urls were included in the page here: https://github.com/adnanh/webhook/wiki/Hook-Examples

It would help newbies such as myself to learn :)

**Comment by bbarclay** [comment]
Yes, I 100% agree. It's so hard to follow too, because you can't see the sh scripts so you have no idea what is applying to where. Also, there doesn't seem to be any simple way to just pass a value in a get request, into an sh file. What makes it challenging too, is that if you search the internet for "Webhook" or webhooks, it's such a generic term that there is very little documentation. For instance if I have an shell script with a variable in it. $1. How do I make a get call of /hooks/id?$variable="something" into the sh script. I've tried but with no success. And there doesn't seem to be much documentation around that. Being that the example don't show the sh scrips and they don't show the urls. It's tough to figure out how to make this happen, with even a post request. What would be super cool. is some basic scripts and a few postman examples to download, so you can automatically start interacting with this to see how it even works.

**Comment by moorereason** [comment]
@bbarclay, thanks for the feedback.  If you have specific questions about how to use `webhook`, use the [discussions area](https://github.com/adnanh/webhook/discussions) for that.
